.pre-commit-hooks.yaml

```
@@ -1,6 +1,16 @@
--   id: check
-    name: FawltyDeps
-    description: Python dependency checker
-    entry: fawltydeps
+-   id: check-undeclared
+    name: FawltyDeps-undeclared
+    description: "FawltyDeps: Check for undeclared dependencies"
+    entry: fawltydeps --check-undeclared
     language: python
+    files: ".*\\.py|ipynb"
+    args:
+      - --code
+    pass_filenames: true
```

- Instead of `files`, I think we can use the `types_or` directive which yields a more readable result. At least this seems to work in my local testing.
- For `args`, I prefer the single-line style when there's only one item
- And I think we can skip `pass_filename` since its default value is already `true`
- Finally, a blank line between the two hooks/checks makes the YAML a bit easier to read, IMHO.

```suggestion
    args: [--code]
    types_or: [python, jupyter]

```